streams/src/main/java/org/apache/kafka/streams/kstream/WindowedSerdes.java

```diff
@@ -30,6 +30,10 @@ public TimeWindowedSerde() {
         public TimeWindowedSerde(final Serde<T> inner) {
             super(new TimeWindowedSerializer<>(inner.serializer()), new TimeWindowedDeserializer<>(inner.deserializer()));
         }
+
+        public TimeWindowedSerde(final Serde<T> inner, final long windowSize) {
```

Comment:
This is a public API change and requires a KIP (cf. https://cwiki.apache.org/confluence/display/KAFKA/Kafka+Improvement+Proposals)

Reply:
@kamalcph Though we want to fix KAFKA-4468 to allow window size be passed into the time windowed deserializers, I'd suggest do it in a separate PR and keep this a smaller scope of bug fixes only. Otherwise this PR will drag long since it takes much more time for KIP discussions.

Also note that there is already a KIP related to this function that is underway: https://cwiki.apache.org/confluence/display/KAFKA/KIP-300%3A+Add+Windowed+KTable+API+in+StreamsBuilder

cc @abbccdda who worked on KIP-300.

Reply:
Removed the public API change.

Reply:
@kamalcph This still exist in the PR, did you forget to push your commit?

Reply:
Fixed it.